bitcoin/src/address/mod.rs

```diff
@@ -874,7 +875,7 @@ mod tests {
         let key = "033bc8c83c52df5712229a2f72206d90192366c36428cb0c12b6af98324d97bfbc"
             .parse::<CompressedPublicKey>()
             .unwrap();
-        let addr = Address::p2wpkh(&key, Bitcoin);
+        let addr = Address::p2wpkh(&key, KnownHrp::Mainnet);
```

Comment:
IMO these tests could've stayed but whatever...

Reply:
Oh, you are right. I did the change before I used `impl Into`.